docs/configurations.md

```diff
@@ -275,3 +275,33 @@ from sqladmin import Admin
 
 admin = Admin(templates_dir="my_templates", ...)
 ```
+
+
+## Custom Action
+
+To add custom action on models to the Admin interface, you can use the `action` annotation.
+
+For example:
+
+!!! example
+
+    ```python
+    from sqladmin import BaseView, action
+
+    class UserAdmin(ModelView, model=User):
+    
+        @action(name='approve_user',label="Approve", confirmation_message="Are you sure ?", add_in_detail=True, add_in_list=True)
```

Comment:
Thanks again for the PR,
After taking a look into what Flask-Admin does I have some ideas:

- In the list page, we should be able to select multiple records and apply apply this action to batch of items, so I think both UI and Backend should change for that.
- I think it'd be easier to review if we separate detail and list PRs
- I think we should either some modal, or call that URL with Ajax, I think the redirect is not much useful and redirecting back from that URL to Admin can be hard.
- I think we should update `actions` to re-use existing code before merging.